addons/my-ocular/addon.json

```diff
@@ -31,8 +32,8 @@
   ],
   "userstyles": [
     {
-      "url": "discuss.css",
-      "matches": ["https://scratch.mit.edu/discuss/topic/*"],
+      "url": "status-dot.css",
+      "matches": ["topics", "https://scratch.mit.edu/users/*/"],
       "if": {
         "settings": { "discuss": true }
       }
```

Comment:
Disabling CSS in profiles if `SHOW STATUSES ON FORUMS` is disabled, but `SHOW STATUSES ON PROFILES` is enabled? Looks like no one noticed this!